src/utils/formatting.py

```
@@ -0,0 +1,258 @@
+    to simple format like '2025-06-03 20:23:43'
+
+    Args:
+        dt: datetime object or datetime string
+
+    Returns:
+        Formatted datetime string in simple format
+    """
+    if isinstance(dt, datetime):
+        # It's a datetime object
+        return dt.strftime("%Y-%m-%d %H:%M:%S")
+    elif isinstance(dt, str):
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)